prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs

```diff
@@ -209,5 +210,22 @@ protected override Task<bool> ProcessRuleInternal(Enrollee enrollee)
                 return Task.FromResult(passed);
             }
         }
+
+
+        // Check if the enrollee has the AlwaysManual flag set
+        public class AlwaysManualRule : BaseAutomaticAdjudicationRule
+        {
+            protected override Task<bool> ProcessRuleInternal(Enrollee enrollee)
+            {
+                var passed = true;
+                if (enrollee.AlwaysManual == true)
+                {
+                    AddReason(enrollee, StatusReason.ALWAYS_MANUAL_CODE);
+                    passed = false;
+                }
+
+                return Task.FromResult(passed);
+            }
```

Comment:
This can be made much simpler since the rule is just the negation of a field.
            ```
protected override Task<bool> ProcessRuleInternal(Enrollee enrollee)
{
    if (enrollee.AlwaysManual)
    {
        AddReason(enrollee, StatusReason.ALWAYS_MANUAL_CODE);
    }

    return Task.FromResult(!enrollee.AlwaysManual);
}
```